irr/include/SSkinMeshBuffer.h

```diff
@@ -368,15 +348,11 @@ struct SSkinMeshBuffer : public IMeshBuffer
 	//! Call this after changing the positions of any vertex.
 	void boundingBoxNeedsRecalculated(void) { BoundingBoxNeedsRecalculated = true; }
 
-	std::vector<video::S3DVertexTangents> Vertices_Tangents;
-	std::vector<video::S3DVertex2TCoords> Vertices_2TCoords;
-	std::vector<video::S3DVertex> Vertices_Standard;
-	std::vector<u16> Indices;
+	SVertexBufferTangents *Vertices_Tangents;
+	SVertexBufferLightMap *Vertices_2TCoords;
+	SVertexBuffer *Vertices_Standard;
+	SIndexBuffer *Indices;
```

Comment:
We might need to define or explicitly delete the move and copy constructors so that these pointers aren't cloned by accident.

Reply:
apparently Irrlicht hasn't done this before but it's easy, here you go: efa836cc7